### PR TITLE
[frontend/backend] Allow color customization for banner messages (#2508)

### PR DESCRIPTION
**Pull request by Kedae**

### CODE REVIEW

**Comment by marieflorescontact** [comment]
Locally tested 👌

File: opencti-platform/opencti-front/src/private/components/settings/settings_messages/SettingsMessages.tsx [hunk 1]
```diff
@@ -65,11 +66,36 @@ const SettingsMessages = ({
...
+          <div
+            style={{
+              backgroundColor,
+              borderLeft,
+              color,
+              textAlign: 'center',
+              fontWeight: 500,
+            }}
+          >
+            Sample
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/settings/settings_messages/SettingsMessages.tsx]: No translation?

**Comment by marieflorescontact** [comment]
Sample witdth could be smaller (more space before message)
Capture d'écran 2023-07-06 105910

---
Done

**Comment by Archidoit** [comment]
Create a message, enter 'red' in the color field of the form, click on the 'Create' button -> an error occurs and the whole Parameters tab is blocked

---
Done

File: opencti-platform/opencti-front/src/private/components/settings/settings_messages/SettingsMessages.tsx (hunk 1 above)
Comment (Kedae) [reply opencti-platform/opencti-front/src/private/components/settings/settings_messages/SettingsMessages.tsx]: Done

**Comment by marieflorescontact** [comment]
Locally tested 👌

**Review by marieflorescontact — APPROVED** [review]

**Pull request merged**